### DISCUSSION TITLE
How to initialize REDE pin locally?

### DISCUSSION THREAD

**Discussion by Thermelgy-Repo** [eModbus/eModbus#319] [Q&A] [answered]
I want to initialize the REDE pin locally other than initializing while creating the global objects. Any help is appreciable.

**Exisiting Method:**

```
#include <Arduino.h>
// Include the header for the ModbusClient RTU style
#include "ModbusClientRTU.h"  

ModbusClientRTU MB(REDEPIN); // REDEPin initializing globally


```

**Comment by Miq1** [comment, marked as answer]
So you want to avoid the ``pinMode()`` call in the constructor? This is be possible if you will write your own ``RTScallback`` function to handle the pin. In this case, nothing is done for any pin at all. Please see the [docs about it here](https://emodbus.github.io/modbusclient-rtu-api#modbusclientrturtscallback-func-and-modbusclientrturtscallback-func-uint16_t-queuelimit).
